Sprint-3/quote-generator/quotes.js

```diff
@@ -1,3 +1,18 @@
+const newQuoteBtn = document.getElementById("new-quote");
+const quoteBlock = document.getElementById("quote");
+const authorBlock = document.getElementById("author");
+
+function displayQuote() {
+  let chosenQuoteObject = pickFromArray(quotes);
+  let chosenQuoteArray = Object.entries(chosenQuoteObject);
+  let chosenQuoteString = chosenQuoteArray[0][1];
+  let chosenAuthorString = chosenQuoteArray[1][1];
+  quoteBlock.textContent = `"${chosenQuoteString}"`;
+  authorBlock.textContent = `${chosenAuthorString}`;
```

Comment:
`chosenQuoteObject` is an object with two keys, "author" and "quote". There is no need to convert them to arrays.
And since you are using the value of author (which is a string) directly as the text content of `authorBlock`, you can just write (without lines 7-9):
```
quoteBlock.textContent = `"${chosenQuoteObject.quote}"`;
authorBlock.textContent = chosenAuthorString;
```

Reply:
Thank you @cjyuan 
That's a great point. I really don't need to convert this object to an array. I've made changes to my code. Thanks again!